lib/foreman_maintain.rb

```diff
@@ -140,6 +140,27 @@ def storage(label = :default)
       logger.error "Invalid Storage label i.e #{label}. Error - #{e.message}"
     end
 
+    def pkg_and_cmd_name
+      if package_manager.installed?(['satellite'])
+        return %w[satellite-maintain satellite-maintain]
+      end
+
+      %w[rubygem-foreman_maintain foreman-maintain]
+    end
```

Comment:
As we have added Capsule support, instead of just checking satellite package could you please do something like -

```
def pkg_and_cmd_name
      instance_feature = ForemanMaintain.available_features(:label => :instance).first
      if instance_feature.downstream
        return %w[satellite-maintain satellite-maintain]
      end

      %w[rubygem-foreman_maintain foreman-maintain]
    end
```